### DISCUSSION TITLE
Are all OSVDB adversaries included in your database or just the ones associated with a CVE number?

### DISCUSSION THREAD

**Discussion by jasnow** [cloudsecurityalliance/gsd-tools#169] [Q&A]
Are all OSVDB adversaries included in your database or just the ones associated with a CVE number?

**Comment by kurtseifried** [comment]
Currently we only ingest CVE, but OSVDB, GHSA and several others are on the todo list for this year. We're working on normalizing the data format first. Especially because we have namespaces we can add lots of sources easily, and deduplicate them.
